examples/transmitter/swagger_server/swagger/swagger.yaml

```diff
@@ -840,6 +860,31 @@ components:
             Stream Management API calls that require authorization.
       example:
         token: 49e5e7785e4e4f688aa49e2585970370
+    TriggerEventParameters:
+      title: Trigger Event Parameters
+      required:
+      - event_type
+      - subject
+      type: object
+      properties:
+        event_type:
```

Comment:
The type of this should be a OneOf so we can limit the possible strings to the event type names

Reply:
it's a string whose model currently is:
```
event_type: str = Field(
        ...,
        description='Read-Write.\nSupports all  (not yet supported)[RISC](https://openid.net/specs/openid-risc-event-types-1_0-ID1.html) and (supported)[CAEP](https://openid.net/specs/openid-caep-specification-1_0-ID1.html) event types.      ',
        example='credential-compromise',
    )
```
if i change it to:
```
event_type:
    OneOf:
      - type: string
``` 
the model changes to:
event_type: Any = Field(
        ...,
        description='Supports all  (not yet supported)[RISC](https://openid.net/specs/openid-risc-event-types-1_0-ID1.html) and (supported)[CAEP](https://openid.net/specs/openid-caep-specification-1_0-ID1.html) event types.',
        example='credential-compromise',
    )
    ```